espnet/nets/pytorch_backend/rnn/decoders_transducer.py

```diff
@@ -343,10 +326,24 @@ def __init__(self, eprojs, odim, dtype, dlayers, dunits, blank, att,
         self.ignore_id = -1
         self.blank = blank
 
-    def zero_state(self, h_pad):
-        return h_pad.new_zeros(h_pad.size(0), self.dunits)
+    def zero_state(self, ey):
+        z_list = [ey.new_zeros(ey.size(0), self.dunits)]
+        c_list = [ey.new_zeros(ey.size(0), self.dunits)]
+
+        for _ in six.moves.range(1, self.dlayers):
+            z_list.append(ey.new_zeros(ey.size(0), self.dunits))
+            c_list.append(ey.new_zeros(ey.size(0), self.dunits))
+
+        return z_list, c_list
+
+    def rnn_forward(self, ey, dstate):
```

Comment:
ditto

Reply:
Done. Feel free to tell me if you think I should add more comments/doc!

Reply:
Cool. You almost added docstrings to everything public in this module. Maybe, you can remove this file from the blacklist 
https://github.com/espnet/espnet/blob/f9e8f676be21d530b473384fb8c45afb17396be4/ci/test_flake8.sh#L75

for [flake8-docstrings](https://pypi.org/project/flake8-docstrings/) by a few more changes.

Reply:
What remaining method needs a docstring? Does I need to add a blank docstring under __init__ and decoder_for?

Reply:
Done. One question though : what are the conditions for a first line to end up with `D401: First line should be in imperative mood`?
I'm confused by the fact that `"""RNN forward.` didn't trigger the error in rnn_forward() but did trigger for `Decoder forward.` in `forward()`...

Reply:
Wonderful!! The D401 seems to have high false positive than false negative. Maybe, its classifier is not enough smart.